README.md

```diff
@@ -1,18 +1,18 @@
 # Lovelace Valetudo Map Card
 
-Draws the map available from a Xiaomi Vacuum cleaner flashed with [Valetudo](https://github.com/Hypfer/Valetudo) in a [Home Assistant](https://www.home-assistant.io/) Lovelace card.
+Draws the map that is provided by [ICantBelieveItsNotValetudo](https://github.com/Hypfer/ICantBelieveItsNotValetudo) from a Xiaomi vacuum cleaner, that is rooted and flashed with [Valetudo](https://github.com/Hypfer/Valetudo) in a [Home Assistant](https://www.home-assistant.io/) Lovelace card.
 
 ## Configuration 
 
-lovelace.yaml:
+`lovelace.yaml`: Add custom Lovelace configuration in Home Assistant to enable valetudo-map-card JavaScript. Go to your HA overview, then right top "Configure UI", then right top again on the 3 dots and "Raw config editor".
 ```yaml
 resources:
   - type: js
     url: /community_plugin/lovelace-valetudo-map-card/valetudo-map-card.js
 ```
 
-configuration.yaml:
-Valetudo officially supports MQTT, with the preferred example configuration as follows. Unfortunately Home Assistant does not support authentication via MQTT. See below for an alternative example configuration using Valetudo's REST API (unsupported) if you prefer not to use MQTT or require authentication.
+`configuration.yaml`:
+[Valetudo officially supports MQTT](https://github.com/Hypfer/Valetudo/wiki/Home-Assistant-Integration), with the preferred example configuration as follows. Unfortunately Home Assistant does not support authentication via MQTT. See below for a deprecated alternative example configuration using Valetudo's REST API (unsupported) if you prefer not to use MQTT or require authentication.
```

Comment:
The first paragraph is indeed useful. Sadly, it also tells you to use ICantBelieveItsNotValetudo, which is incorrect for this addon. So we can't really link to this page as-is.

Reply:
Fair enough, link removed.